src/components/EmptyState/EmptyState.jsx

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright (C) 2018 The Trustees of Indiana University
+SPDX-License-Identifier: BSD-3-Clause
+*/
+import * as React from "react";
+import Actions from "./Actions";
+import Content from "./Content";
+
+const classPrefix = "rvt-empty-state";
+
+// https://dev.to/hey_yogini/create-react-subcomponents-in-a-simple-way-5h1f
+
+const EmptyState = ({ children, ...attrs }) => {
+  let subComponentList = Object.keys(EmptyState);
+
+  let subComponents = React.Children.map(children, (child) =>
```

Comment:
Restricting the children to only the Actions or Content components seems like an unnecessary restriction, which may cause frustration if the user needs to do something a little different — like even including an extra `div` or something. It should be on the user to use the components in the right way. I recommend to just let the children pass through untouched.